example/model-parallel/lstm/README.md

```diff
@@ -1,4 +1,13 @@
 Model Parallel LSTM
 ===================
+
 This is an example showing how to do model parallel LSTM in MXNet.
-Most of the code is duplicated with the rnn example, and should be eventually merged.
+
+To run this example, first make sure you download a dataset of PenTreeBank available
```

Comment:
nitpik: We use the [PenTreeBank dataset](link)  in this example. Download the dataset with below command.